scripts/contributing.js

```diff
@@ -0,0 +1,36 @@
+/* global jQuery*/
+
+(function($){
+    var githubIssuesEndpoint = 'https://api.github.com/repos/babel/babel/issues';
+
+    $.ajax(githubIssuesEndpoint, {
+            data : {
+                labels : 'beginner-friendly'
+            },
+            success : function(data){
+                for(var i = 0; i < data.length; i++){
+                    $('.beginnerFriendlyIssues').append('<li><a href="' + data[i].html_url + '"><span class="issueNumber">#' + data[i].number +'</span></a><span>' + data[i].title + '</span></li>');
+                }
+            },
+            error : function(xhr, status, error){
+                throw error;
+            }
+        }
+    );
+
+    $.ajax(githubIssuesEndpoint, {
+            data : {
+                labels : 'help wanted'
+            },
+            success : function(data){
+                for(var i = 0; i < data.length; i++){
+                    $('.helpWantedIssues').append('<li><a href="' + data[i].html_url + '"><span class="issueNumber">#' + data[i].number +'</span></a><span>' + data[i].title + '</span></li>');
+                }
+            },
+            error : function(xhr, status, error){
+                throw error;
+            }
+        }
+    );
```

Comment:
Using more than one label seems to restrict the query more by only including issues with both "beginner-friendly" and "help wanted". So that url only provides 2 issues while this url:

    https://api.github.com/repos/babel/babel/issues?labels=help%20wanted

provides 9 issues.

I don't think we'll be able to combine these.